browser/src/history.js

```diff
@@ -103,38 +104,6 @@ class History{
     this.formatButtons();
   }
 
-  /**
-   * Initializes the action history using the history stored on the backend
-   * @param {*} actionFrames frame of each action performed on the project
-   * @param {*} initFrame frame displayed at the bottom of the undoStack
-   * @param {*} finalFrame frame displayed at the top of the undoStack
-   */
-  initializeHistory(actionFrames, initFrame = 0, finalFrame = current_frame) {
-      // Initialize undoStack to with actions recorded on backend
-      let prevFrame = initFrame;
-      for (let frame of actionFrames) {
```

Comment:
Maybe I'm missing it elsewhere, but did this get moved or just removed?

Reply:
This was intended to let users to undo edits from past sessions or when reloading the past, but it breaks when editing labels on different features, so I removed it to avoid the bug. The issue is detailed in #220. I'm open to getting it working again, removing it, or leaving it in with the bug. I don't think we have any annotations under way that use multiple features, so I don't think it's a bug that would affect annotation right now